src/main/java/UofTeams.java

```diff
@@ -0,0 +1,187 @@
+        TimerInteractor timerInteractor = new TimerInteractor(timerDataAccess, deletePostInteractor);
+        TimerController timerController = new TimerController(timerInteractor);
+         */
+
+        // initialize stuff for log in
+        LogInViewModel logInViewModel = new LogInViewModel();
+        LogInOutputBoundary logInPresenter = new LogInPresenter(logInViewModel);
+        LogInDsGateway logInDatabaseAccess = new LogInDatabaseAccess(usersFilePath);
+        LogInInputBoundary logInInteractor = new LogInInteractor(logInDatabaseAccess, logInPresenter);
+        LogInController logInController = new LogInController(logInInteractor);
+        LogInView logInView = new LogInView(logInController);
+
+        // initialize stuff for log out
+        LogOutViewModel logOutViewModel = new LogOutViewModel();
+        LogOutOutputBoundary logOutPresenter = new LogOutPresenter(logOutViewModel);
+        LogOutInputBoundary logOutInteractor = new LogOutInteractor(logOutPresenter);
+        LogOutController logOutController = new LogOutController(logOutInteractor);
+        LogOutView logOutView = new LogOutView(logOutController);
+
+        // initialize stuff for favourite_post
+        /*
+        UserFactory userFactory = new UserFactory();
+        FavourtiteViewModel favourtiteViewModel = new FavouriteViewModel();
+        FavouritePresenter favouritePresenter = new FavouritePresenter(favourtiteViewModel);
+        DataAccess dataAccess = new DataAccess(postFactory, userFactory, generalPath);
+        FavouriteInteractor favouriteInteractor = new FavouriteInteractor(dataAccess, favouritePresenter);
+        FavouriteController favouriteController = new FavouriteController(favouriteInteractor);
+         */
+
+        // initialize stuff for view_post
+        ViewPostView viewPostView = new ViewPostView();
+
+        ViewPostViewModel viewPostViewModel = new ViewPostViewModel(viewPostView);
+        ViewPostPresenter viewPostPresenter = new ViewPostPresenter(viewPostViewModel);
+        ViewPostDsGateway viewPostGateway = new ViewPostDatabaseAccess(generalPath);
+        ViewPostInteractor viewPostInteractor = new ViewPostInteractor(viewPostGateway, viewPostPresenter);
+        ViewPostController viewPostController = new ViewPostController(viewPostInteractor);
+
+        PostListView postListView = new PostListView(viewPostController, filterPostBarView);
+        HeaderView headerView = new HeaderView(generalPath);
+
+
+        // initialize main view
+        MasterLandingView masterLandingView = new MasterLandingView(signUpView, logInView);
+        MainPostView mainPostView = new MainPostView(viewPostView, postListView, headerView);
+        MainFrame mainFrame = new MainFrame(masterLandingView, mainPostView);
+
+        // Setting up observers
+        // filterPostViewModel.addObserver(postListView);
+        makePostViewModel.addObserver(makePostView);
+
```

Comment:
Do you think it would be okay if I did this for FavouriteView in the MainPostView in the view_post ui?
The Favourite View gets initialized every time the view gets refreshed so I can't handle it here. 
HOWEVER if that is a bad idea, I can also create a setter for the FavouriteView that will pass new post IDs to the view every time ViewPostView is refreshed